baremetal/metal3machine_manager_test.go

```diff
@@ -78,26 +78,26 @@ func bmmSpecAll() *capm3.Metal3MachineSpec {
 	}
 }
 
-func bmmSecretStatus() *capm3.Metal3MachineStatus {
+func m3mSecretStatus() *capm3.Metal3MachineStatus {
 	return &capm3.Metal3MachineStatus{
 		UserData: &corev1.SecretReference{
-			Name:      "mybmmachine-user-data",
+			Name:      "mym3machine-user-data",
 			Namespace: "myns",
 		},
 		MetaData: &corev1.SecretReference{
-			Name:      "mybmmachine-meta-data",
+			Name:      "mym3machine-meta-data",
 			Namespace: "myns",
 		},
 		NetworkData: &corev1.SecretReference{
-			Name:      "mybmmachine-network-data",
+			Name:      "mym3machine-network-data",
 			Namespace: "myns",
 		},
 	}
 }
 
 func consumerRef() *corev1.ObjectReference {
 	return &corev1.ObjectReference{
-		Name:       "mybmmachine",
+		Name:       "mym3machine",
 		Namespace:  "myns",
 		Kind:       "BMMachine",
```

Comment:
Should the value of ` Kind:` be kept as is?

Reply:
Should it be M3Machine?

Reply:
thanks, I replaced only bmm, so I updated the patch to change BMM also